contracts/prize-pool/PrizePool.sol

```diff
@@ -421,6 +421,13 @@ abstract contract PrizePool is OwnableUpgradeSafe, BaseRelayRecipient, Reentranc
     }
   }
 
+  /// @notice Allows the Governor to add Controlled Tokens to the Prize Pool
+  /// @param _controlledToken The address of the Controlled Token to add
+  function addControlledToken(address _controlledToken) external onlyOwner {
+    require(ControlledToken(_controlledToken).controller() == this, "PrizePool/token-ctrlr-mismatch");
+    _tokens.addAddress(_controlledToken);
```

Comment:
Also make sure it doesn't already contain it!

Reply:
handled in `addAddress`